indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -897,7 +907,9 @@
          [(get indexes (or target-index-key :variables))])
 
        :service
-       [(get indexes (or target-index-key :services))]
+       (if all-revisions-index?
+         [(get indexes :all-services-revisions)]
```

Comment:
I see all-service-revisions as the name of the index and all-service**s**-revisions here. We should be consistent.

Reply:
Done.